files/usr/lib/jeos-firstboot

```diff
@@ -181,11 +185,11 @@ if [ -z "$LOCALE_PRESET" ]; then
         run langset.sh $locale || warn $"Setting the locale failed"
     fi
 else
-    locale="$LOCALE_PRESET"
+    locale="$JEOS_LOCALE"
```

Comment:
`$locale` can be replaced by `$JEOS_LOCALE` everywhere.

Reply:
Not sure that this is correct. `$locale` can come from two different paths and later is used in `langset.sh` later on.

Reply:
Ok. The refactor is now in the last PR